week03/LAB/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,26 @@
+/** @jsxImportSource @emotion/react */
+
+import { MENU } from "../../constants/Menu";
+import { Button, Title, Wrapper, ButtonWrapper } from "./Header.styles";
+
+export default function Header({ menu, handleMenu }) {
+  return (
+    <Wrapper>
+      <Title>⚾ 숫자야구 || 깃허브 검색 🔍</Title>
+      <ButtonWrapper>
+        <Button
+          isActive={menu === MENU.GITHUB}
+          onClick={() => handleMenu(MENU.GITHUB)}
+        >
+          깃허브 검색🔍
+        </Button>
+        <Button
+          isActive={menu === MENU.BASEBALL}
+          onClick={() => handleMenu(MENU.BASEBALL)}
+        >
+          숫자 야구⚾
+        </Button>
+      </ButtonWrapper>
+    </Wrapper>
+  );
+}
```

Comment:
```suggestion
const menuList = [
  { key: MENU.GITHUB, label: "깃허브 검색🔍" },
  { key: MENU.BASEBALL, label: "숫자 야구⚾" },
];

export default function Header({ menu, handleMenu }) {
  return (
    <Wrapper>
      <Title>⚾ 숫자야구 || 깃허브 검색 🔍</Title>
      <ButtonWrapper>
        {menuList.map(({ key, label }) => (
          <Button
            key={key}
            isActive={menu === key}
            onClick={() => handleMenu(key)}
          >
            {label}
          </Button>
        ))}
      </ButtonWrapper>
    </Wrapper>
  );
}
```

이런 방법으로 코드 중복을 줄일 수 있을 것 같은데 어떻게 생각하시나요?? 지금은 버튼 두개가 거의 동일한 코드 구조를 가지고 있어서 중복된 코드가 많아보여요